tests/bluetooth/df/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DF_SUPPORT
+	bool
+	default y
```

Comment:
I don't understand why this is needed, isn't it enough to enable it in the prj.conf file?

Reply:
Without it BT_CTLR_DF_SUPPORT will be enabled when nrf52833 board is selected.
When compile UT  the nrf52_bsim board is selected, so no DF code is compiled in controller.
Also this is hidden entry, it will generate warnings during build when selected in prj.conf (correct me if I'm wrong).